src/libasr/intrinsic_func_registry_util_gen.py

```diff
@@ -98,6 +98,12 @@
             "ret_type_arg_idx": 3
         }
     ],
+    "Scan": [
```

Comment:
Let's use some more descriptive name:
```suggestion
    "StringFindSet": [
```